kde-apps/kmahjongg/kmahjongg-24.12.3.recipe

```diff
@@ -0,0 +1,162 @@
+	lib:libKF6KIOCore$secondaryArchSuffix
+	lib:libKF6NewStuffCore$secondaryArchSuffix
+	lib:libKF6NewStuffWidgets$secondaryArchSuffix
+	lib:libKF6Package$secondaryArchSuffix
+	lib:libKF6Service$secondaryArchSuffix
+	lib:libKF6Solid$secondaryArchSuffix
+	lib:libKF6TextEditor$secondaryArchSuffix
+	lib:libKF6TextWidgets$secondaryArchSuffix
+	lib:libKF6WidgetsAddons$secondaryArchSuffix
+	lib:libKF6WindowSystem$secondaryArchSuffix
+	lib:libKF6XmlGui$secondaryArchSuffix
+	# Qt6
+	lib:libQt6Core$secondaryArchSuffix
+	lib:libQt6DBus$secondaryArchSuffix
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Qml$secondaryArchSuffix
+	lib:libQt6Svg$secondaryArchSuffix
+	lib:libQt6SvgWidgets$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	extra_cmake_modules$secondaryArchSuffix
+	qt6_tools$secondaryArchSuffix
+	devel:libkdegames6$secondaryArchSuffix
+	devel:libKMahjongg6$secondaryArchSuffix
+	# KF6
+	devel:libKMahjongg6$secondaryArchSuffix
+	devel:libKF6Archive$secondaryArchSuffix
+	devel:libKF6Bookmarks$secondaryArchSuffix
+	devel:libKF6Codecs$secondaryArchSuffix
+	devel:libKF6ColorScheme$secondaryArchSuffix
+	devel:libKF6Completion$secondaryArchSuffix
+	devel:libKF6ConfigCore$secondaryArchSuffix
+	devel:libKF6ConfigWidgets$secondaryArchSuffix
+	devel:libKF6CoreAddons$secondaryArchSuffix
+	devel:libKF6Crash$secondaryArchSuffix
+	devel:libKF6DBusAddons$secondaryArchSuffix
+	devel:libKF6DocTools$secondaryArchSuffix
+	devel:libKF6GuiAddons$secondaryArchSuffix
+	devel:libKF6I18n$secondaryArchSuffix
+	devel:libKF6IconThemes$secondaryArchSuffix
+	devel:libKF6ItemModels$secondaryArchSuffix
+	devel:libKF6ItemViews$secondaryArchSuffix
+	devel:libKF6JobWidgets$secondaryArchSuffix
+	devel:libKF6KIOCore$secondaryArchSuffix
+	devel:libKF6NewStuffCore$secondaryArchSuffix
+	devel:libKF6NewStuffWidgets$secondaryArchSuffix
+	devel:libKF6Package$secondaryArchSuffix
+	devel:libKF6Service$secondaryArchSuffix
+	devel:libKF6Solid$secondaryArchSuffix
+	devel:libKF6TextEditor$secondaryArchSuffix
+	devel:libKF6TextWidgets$secondaryArchSuffix
+	devel:libKF6WidgetsAddons$secondaryArchSuffix
+	devel:libKF6WindowSystem$secondaryArchSuffix
+	devel:libKF6XmlGui$secondaryArchSuffix
+	# Qt6
+	devel:libQt6Core$secondaryArchSuffix
+	devel:libQt6Svg$secondaryArchSuffix
+	devel:libQt6SvgWidgets$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:msgfmt$secondaryArchSuffix
+	cmd:msgmerge$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+PATCH()
+{
+	cd $sourceDir
+	# disable docs generation
+	sed -e '/add_subdirectory(doc)/ s/^#*/#/' -i CMakeLists.txt
+	sed -e '/kdoctools_install(po)/ s/^#*/#/' -i CMakeLists.txt
+	# remove dbus service
+	sed -e '/KDBusService service;/d' -i src/main.cpp
+}
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+
+	cmake .. \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX=$appsDir \
```

Comment:
You can use `$cmakeDirArgs` here for the rest and `-DCMAKE_INSTALL_BINDIR=$appsDir` for the binary, saves removing $a`ppsDir/bin` later.